persistence/persistence-mybatis-test/src/test/java/org/cbioportal/persistence/mybatis/CancerTypeMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,111 @@
+package org.cbioportal.persistence.mybatis;
+
+import org.cbioportal.model.TypeOfCancer;
+import org.cbioportal.model.meta.BaseMeta;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("/testContextDatabase.xml")
+@Configurable
+public class CancerTypeMyBatisRepositoryTest {
+
+    @Autowired
+    private CancerTypeMyBatisRepository cancerTypeMyBatisRepository;
+
+    @Test
+    public void getAllCancerTypesIdProjection() throws Exception {
+
+        List<TypeOfCancer> result = cancerTypeMyBatisRepository.getAllCancerTypes("ID", null, null, null, null);
+
+        Assert.assertEquals(2, result.size());
+        TypeOfCancer typeOfCancer = result.get(0);
```

Comment:
I think that is too much implicit information. In this case, I would like to recommend adding a comment line in the code explaining this or implementing something like a sorting to make things explicit. Otherwise the next developer that looks at this code would have to figure this out all over again when in doubt...